mimic/rest/noit_api.py

```diff
@@ -0,0 +1,114 @@
+# -*- test-case-name: mimic.test.test_noit -*-
+"""
+Defines get token, impersonation
+"""
+import xmltodict
+from uuid import UUID
+from twisted.web.server import Request
+from mimic.rest.mimicapp import MimicApp
+from mimic.canned_responses.noit import (create_check, get_check,
+                                         get_all_checks, delete_check,
+                                         test_check)
+
+
+# TO DO:
+# Move away from using dict for the templates
+# Include all check type to the metrics in the fixtures
+# Metrics Fixture to populate as per test check request
+# Include more error cases based on ele's tests
+
+
+Request.defaultContentType = 'application/xml'
```

Comment:
This changes the default content type of *all* HTTP responses in this entire process.  Instead, just set the `application/xml` content type explicitly in each place you're returning a response.